src/index.js

```diff
@@ -164,7 +164,7 @@ function serializeHandler(handler) {
 }
 
 module.exports = {
-  onPostBuild: async ({ inputs: { sourceDir }, constants, utils }) => {
+  onPostBuild: async ({ constants: { IS_LOCAL, NETLIFY_API_TOKEN, EDGE_HANDLERS_SRC: sourceDir }, utils }) => {
```

Comment:
i'd be ok with keeping `EDGE_HANDLERS_SRC` as the variable name

Reply:
Should it be changed only in this function, or also in the other functions using `sourceDir` in this file?

Reply:
I like the consistency and clarity of using `EDGE_HANDLERS_SRC` across the functions since that variable never really changes

Reply:
i don't care a lot either way

Reply:
Fixed!